appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -785,7 +785,8 @@ public void execute() {
 
       VerticalPanel DialogBoxContents = new VerticalPanel();
       String html = MESSAGES.gitBuildId(GitBuildId.getDate(), GitBuildId.getVersion()) +
-          "<BR/>Use Companion: " + BlocklyPanel.getCompVersion();
+          "<BR/>Use Companion: " + BlocklyPanel.getCompVersion() +
+          "<BR/>Target Android SDK: " + YaVersion.TARGET_SDK_VERSION;
```

Comment:
We should add these strings (sans `<br>` tag) to OdeMessages so they can be translated. We were also asked to consider including the corresponding Android version number (e.g., "Android 9.0 Pie") as well as not all of our users will know the mapping of SDK version to marketing version. Add that as a string in YaVersion and include it here.

Reply:
I implemented the requested changes in the latest commits